src/xcode/ENA/ENA/Source/Scenes/TicketValidation/SecondConsent/SecondTicketValidationConsentViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+class SecondTicketValidationConsentViewController: DynamicTableViewController, FooterViewHandling {
```

Comment:
```suggestion
class TicketValidationSecondConsentViewController: DynamicTableViewController, FooterViewHandling {
```
Since its not the Second Ticket Validation but the Second Consent in the (first/only) Ticketvalidation

Reply:
I think we could keep it like it is, because it is the second ticket-validation-consent. I used this order for the first ticket validation consent already, because you would not say "Please sign the ticket validation first consent!" but rather "Please sign the first ticket validation consent!". So for better readability I put "first" first 🤓